### PR TITLE
fix: fix incorrect label passed to `ipc` protocol

### LINKED ISSUE #11171: [bug] broken IPC in multi window app on Linux
### Describe the bug

In a multi-window Tauri app on Linux, only the first window ever created can receive events and have its `listen` callbacks triggered. All subsequent windows do not receive any events.

In this video, the same Tauri app runs on Windows on the left and Linux on the right.

Every window ran `listen("reply", (event) => {alert(event.payload})})`.
When the button is pressed, a Rust Tauri command is invoked, which emits the `reply` event.

On Windows, the app behaves correctly and every window receives the `reply` event.

On Linux, only the first window ever receives any events.

https://github.com/user-attachments/assets/b2c0fd72-d7b8-4764-a92d-f82a2dd9af7b

### Reproduction

The example shown in the video: https://github.com/Ynng/tauri-listen-test

Or:

- Clone the tauri repo on an Ubuntu 24 machine
- Run `cargo run --example multiwindow`
- Try to send a message to the second/third window

### Expected behavior

All windows should receive events that are "emitted to all targets".

### Full `tauri info` output

```text
[✔] Environment
    - OS: Fedora 40.0.0 x86_64 (X64)
    ✔ webkit2gtk-4.1: 2.44.3
    ✔ rsvg2: 2.57.1
    ✔ rustc: 1.80.1 (3f5fd8dd4 2024-08-06)
    ✔ cargo: 1.80.1 (376290515 2024-07-16)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-x86_64-unknown-linux-gnu (environment override by RUSTUP_TOOLCHAIN)
    - node: 21.7.2
    - pnpm: 9.7.1
    - npm: 10.5.0

[-] Packages
    - tauri 🦀: 2.0.0-rc.11
    - tauri-build 🦀: 2.0.0-rc.13
    - wry 🦀: 0.44.1
    - tao 🦀: 0.30.2
    - tauri-cli 🦀: 2.0.0-rc.10

[-] Plugins
    - tauri-plugin-shell 🦀: 2.0.0-rc.0

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
```

### Stack trace

_No response_

### Additional context

Undoing #11043 fixes this problem, but re-introduces #10981

### PR DESCRIPTION
**Pull request by amrbashir**
closes #11171

### CODE REVIEW

**Comment by lucasfernog** [comment]
needs https://github.com/tauri-apps/wry/pull/1384

File: .changes/incorrect-label-linux-command.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"tauri": "minor:bug"
+"tauri-runtime": "minor:bug"
+"tauri-runtime-wry": "minor:bug"
+---
+
+On Linux, fix commands, that use `Webview` or `WebviewWindow` as an argument, receiving an inccorect webview when using multi webviews.
```
Comment (lucasfernog) [inline .changes/incorrect-label-linux-command.md]:
```suggestion
On Linux, fix commands, that use `Webview` or `WebviewWindow` as an argument, receiving an incorrect webview when using multi webviews.
```

File: .changes/incorrect-label-linux-protocol.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"tauri": "minor:bug"
+"tauri-runtime": "minor:bug"
+"tauri-runtime-wry": "minor:bug"
+---
+
+On Linux, fix custom protocols receiving an inccorect webview label when using multi webviews
```
Comment (lucasfernog) [inline .changes/incorrect-label-linux-protocol.md]:
```suggestion
On Linux, fix custom protocols receiving an incorrect webview label when using multi webviews
```

File: .changes/incorrect-label-linux-command.md (hunk 1 above)
Comment (gschier) [reply .changes/incorrect-label-linux-command.md]: Incorrect* ;)

File: .changes/incorrect-label-linux-command.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+"tauri": "minor:bug"
```
Comment (lucasfernog) [inline .changes/incorrect-label-linux-command.md]: I think we should release it as a patch instead

Comment (lucasfernog) [reply .changes/incorrect-label-linux-command.md]: tauri-runtime and tauri-runtime-wry probably can keep the minor as it's a breaking change

**Comment by lucasfernog** [comment]
needs wry 0.46 - https://github.com/tauri-apps/wry/pull/1387

File: .changes/incorrect-label-linux-command.md (hunk 3 above)
Comment (amrbashir) [reply .changes/incorrect-label-linux-command.md]: yeah tauri could probably be `patch`, updated

Comment (amrbashir) [inline .changes/incorrect-label-linux-command.md]:
```suggestion
"tauri": "patch:bug"
```

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**